main.cc

```diff
@@ -89,4 +90,17 @@ PYBIND11_MODULE(pycolmap, m) {
           py::arg("inlier_mask"),
           py::arg("camera_dict"),
           "Non-linear refinement.");
+    
+    // Generic two view geometry estimation.
+    m.def("two_view_geometry_estimation", &two_view_geometry_estimation,
+          py::arg("points2D1"),
+          py::arg("points2D2"),
+          py::arg("camera_dict1"),
+          py::arg("camera_dict2"),
+          py::arg("max_error_px"),
+          py::arg("min_inlier_ratio"),
+          py::arg("min_num_trials"),
+          py::arg("max_num_trials"),
+          py::arg("confidence"),
```

Comment:
Nice PR, thanks! Should you make these arguments optional by providing default values?

Reply:
Sure thing -- just added a few defaults.